docs/pages/guides/testing-rsc.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Testing React Server Components
+sidebar_title: Testing RSC
+description: Learn about writing unit tests for React and React Native Server Components in Expo.
+platforms: ['android', 'ios', 'web']
+---
+
+> This guide refers to the experimental feature React Server Components which is still in development.
+
+React Server Components (RSC) are a new feature in React that allows you to build components that render on the server and can be hydrated on the client.
+
+Expo has support for testing React Server Components in your project.
+
+## Jest Testing
+
+Unlike client-based Jest testing, React Server Components run in a Node.js environment, meaning you can more closely emulate the server-side rendering environment with Jest on its own.
```

Comment:
```suggestion
React Server Components run in a Node.js environment, unlike a client-based Jest testing environment. This means you can emulate the server-side rendering environment closely with Jest.
```

Let's rephrase the above paragraph for following reasons:
- Starting with a positive note always helps when reading
- Let's split sentences and rephrase the second sentence for brevity